includes/class-wc-gateway-ppec-client.php

```diff
@@ -547,7 +547,7 @@ protected function get_details( $details, $discounts, $rounded_total, $total ) {
 				// Add line item to make up different between WooCommerce
 				// calculations and PayPal calculations.
 				$diff = round( $details['total_item_amount'] - $rounded_total, $decimals );
-				if ( $diff != 0 ) {
+				if ( abs( $diff ) < 0.0001 ) {
```

Comment:
what's the reasoning to choose the number `0.0001` ? Is it mentioned in PPEC doc?